content/docs/09.administrator-guide/02.deployment/04.restack.md

```diff
@@ -1,56 +1,195 @@
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": [
+                    "arn:aws:iam::223746943638:user/connect",
+                    "arn:aws:iam::223746943638:role/connect"
+                ]
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringEquals": {
+                    "sts:ExternalId": "restack"
+                }
+            }
+        }
+    ]
+  }
+```
+
+5. Click the "Next" button and under the "Add permissions" section, add the `AdministratorAccess` policy.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/2685ee99b153f46b38f99607407dfba72c5c6522-2000x1160.png)
+
+6. Click the "Next" button and type `restack` in the "Role name" field. Then, review the IAM role and click on the "Create role" button.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/fb14345d290077195b986cb3931c0d39689dca6f-2000x1255.png)
+
+7. From the IAM > Roles overview, navigate to the role you've just created and copy its ARN.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/e3750c4fb52873a781e328e81c70237d1968d5da-2560x1440.png)
+
+8. Paste the Role ARN to Restack's user interface and validate the setup by clicking on the 'Connect cloud' button. This will validate if Restack can successfully assume the role in your AWS account.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/ef820e88fd8cd5bd64ca24b79f22bc96cf9fd255-2560x1440.png)
+
+If the AWS cloud account has been successfully added and Restack can connect to it, you can proceed to create a cluster and provision your Kestra instance.
+
+
+**How can restack assume the IAM role created in your account?**
+
+- The `sts:AssumeRole` is an AWS Security Token Service (STS) API operation that allows an IAM user or role in one account to assume a role in another AWS account. This allows the user or role to access resources in the second account that they would not normally have access to.
+
+- The `sts:AssumeRole` operation returns a set of temporary security credentials that the user or role can use to access resources in the second account. These credentials include an access key, a secret access key, and a session token, which can be used to make AWS API requests.
+
+
+## Connect your GCP account
+
+Connect your GCP account in a few steps to Restack.
+
+> Requirements
+- Active Google Cloud Organization
+- Active Google project for all Restack related resources
+- Active billing profile associated with the project
+
+1. Create a new service account in your Google project, you created to connect with Restack
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/43f688c517cfca2c2e430d17d0069ed3fb74b2e8-6016x3384.png)
+
+2. Copy connect@restackio.iam.gserviceaccount.com
+
+3. Navigate to https://console.cloud.google.com/ and to your project-id
+
+4. Go to IAM & Admin > Service Accounts
+
+Create a new service account in your project
+Service account name: `restack`
+Service account ID: `restack`
```

Comment:
```suggestion
```

cut as this one conflicting with step 1 which already creates the account?

Reply:
Thanks, I will commit and add clarify in the point 1. that the servcie account and ID should be `restack`